relay-monitors/src/lib.rs

```diff
@@ -78,6 +82,23 @@ pub fn process_check_in(payload: &[u8]) -> Result<Vec<u8>, ProcessCheckInError>
         check_in.status = CheckInStatus::Unknown;
     }
 
+    static SLUG_CLEANER: OnceCell<Regex> = OnceCell::new();
+    let slug_cleaner_regex = SLUG_CLEANER.get_or_init(|| Regex::new(r"[^a-zA-Z0-9\s_-]").unwrap());
```

Comment:
What do you need the `SLUG_CLEANER` for? Why is it `static`? I think what you want to do `static` is `slug_cleaner_regex`?

Same with the `SLUGIFIER` below.